readthedocs/gold/forms.py

```diff
@@ -88,6 +89,18 @@ def __init__(self, *args, **kwargs):
         self.projects = kwargs.pop('projects', None)
         super(GoldProjectForm, self).__init__(*args, **kwargs)
 
+    def clean_project(self):
+        cleaned_data = super(GoldProjectForm, self).clean()
+        project_slug = cleaned_data.get('project', '')
+        project_instance = Project.objects.filter(slug=project_slug)
+
+        # Checking if the project with the entered slug
+        # is absent or present in the database.
+        if not project_instance.exists():
+            raise forms.ValidationError('No project found.')
```

Comment:
We need support translation for this message (using `_(msg)`) (like the above code)

Reply:
like this ?? 
`raise forms.ValidationError(_('No project found.'))`

Reply:
Yeah, that's it

Reply:
thanks.
though i deleted my comment before seeing your comment.